toolkit/tools/imagecustomizer/docs/iso.md

```diff
@@ -2,26 +2,29 @@
 
 ## Overview
 
-Given a full disk image generated by the Azure Linux toolkit, the Azure Linux
-Image Customizer can generate a LiveOS ISO image when the
-`--output-image-format` is set to `iso`.
-
-The LiveOS ISO image is a bootable image that boots into the rootfs of the
-input full disk image. This eliminates the requirement to install that image
-to persistent storage before booting it.
-
-All customizations can still be made to the input full disk image rootfs as
-usual using MIC, and will naturally carry over into the LiveOS ISO. This
-includes customizations for kernel modules, dracut, and other early boot-time
-actions.
-
-While converting the input full disk image into a LiveOS ISO involves copying
+The Azure Linux Image Customizer can customize an input image and package the
+output as a LiveOS iso image. The input image can be a full disk image
+(vhd/vhdx/qcow2/raw) or another LiveOS iso image. The LiveOS iso output format
```

Comment:
does it have to be `LiveOS` iso only? what will happen if a regular iso is used?

Reply:
Correct, the input iso has to an image previously generated by mic. If the input iso does not have the same exact names and location of bits as the one generated by mic, customization will fail.

Reply:
[we need to locate bootloaders, kernel, initrd, rootfs, grub.cfg in specific locations and with specific names]